### PR TITLE
add post-submit

### PR DESCRIPTION
**Pull request by mjudeikis**
@kargakis ptal
This is OSA conformance as post submit job

### CODE REVIEW

**Comment by mjudeikis** [comment]
screenshot from 2018-10-04 08-33-21
@stevekuznetsov @bbguimaraes If this expected? If I run `docker run -it -v $(pwd)/ci-operator:/ci-operator:z registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config/ --to-dir /ci-operator/jobs` I get duplicates in the names of hte files being generated..

**Comment by asalkeld** [comment]
this looks great, but honestly I don't know this  well enough to mark l g t m

**Comment by stevekuznetsov** [comment]
@mjudeikis rebase so you're generating off of latest `master` base

**Comment by mjudeikis** [comment]
@stevekuznetsov still same. But I cant get new image to try too:
```
docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest
Error response from daemon: Get https://registry.svc.ci.openshift.org/v2/ci/determinize-prow-jobs/manifests/latest: unauthorized: authentication required
```

**Comment by stevekuznetsov** [comment]
Weird, let me look into the permissions issue

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,61 @@
 postsubmits:
   openshift/openshift-azure:
+  - agent: kubernetes
+    branches:
+    - master
+    - private-preview-1
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: This should not be here

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+    decorate: true
+    labels:
+      artifacts: images
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: This should not be here

**Pull request closed without merging**